packages/vaadin-combo-box/test/overlay-position.test.js

```diff
@@ -102,15 +103,18 @@ describe('overlay position', () => {
       expect(overlay.getBoundingClientRect().top).to.be.closeTo(input.getBoundingClientRect().bottom, 1);
     });
 
-    it('when the input position width changes overlay width should change', () => {
+    it('when the input position width changes overlay width should change', async () => {
       moveComboBox(xCenter, yBottom, 150);
 
       comboBox.open();
 
-      expect(overlay.getBoundingClientRect().width).to.equal(input.getBoundingClientRect().width);
+      await nextRender(comboBox.$.overlay);
```

Comment:
Could you please point to a change that caused this and other test to be async? It's a bit unclear to me.